app/src/main/java/com/sun/moviesun/data/model/network/NetworkResponseModel.kt

```diff
@@ -0,0 +1,3 @@
+package com.sun.moviesun.data.model.network
+
+open class NetworkResponseModel
```

Comment:
Model là chỉ những class chứa dữ liệu đối tượng thôi e
```suggestion
open class NetworkResponse
```